.github/workflows/ext.yml

```diff
@@ -2,9 +2,9 @@ name: Test External
 
 on:
   push:
-    branches:
-      - main
-  pull_request:
+#    branches:
+#      - main
+#    pull_request:
```

Comment:
These changes are for checking that all tests work in the pipeline, I will remove these changes